wallety/wallet.go

```diff
@@ -0,0 +1,53 @@
+package wallety
+
+import (
+	"math/big"
+	"context"
+	"fmt"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/lightstreams-network/lightchain/authy"
+	"github.com/lightstreams-network/lightchain/txclient"
+	"github.com/lightstreams-network/lightchain/log"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+var logger = log.NewLogger().With("module", "wallety")
```

Comment:
[minor] This `logger` appears to not fit on the logging strategy we implemented from the project. Maybe it is worthy to pass it as an argument on the method you intend to print logs